examples/complete/main.tf

```diff
@@ -70,6 +70,25 @@ module "create_scc_instance" {
   wp_instance_crn                   = module.scc_wp.crn
 }
 
+##############################################################################
+# SCC profile
+##############################################################################
+
+module "create_scc_profile" {
+  source      = "../../modules/profile/."
+  instance_id = module.create_scc_instance.guid
+
+  control_libraries = [
+    {
+      control_library_name    = "IBM Cloud Framework for Financial Services",
+      control_library_version = "1.6.0"
+    }
+  ]
+
+  profile_name        = "${var.prefix}-profile"
+  profile_description = "scc-custom"
```

Comment:
This use case makes no sense? What I would expect is that the code should be creating a custom profile made up of controls. It shouldn't matter about what control library they came from

Reply:
Names are weird for each control, just numbers or `AC-1(B)` etc
<img width="1620" alt="Screenshot 2024-04-18 at 01 02 38" src="https://github.com/terraform-ibm-modules/terraform-ibm-scc/assets/22322183/c748d2ba-2c50-4c60-aa58-8eb1b80f5934">